cmd/helpgen/main.go

```diff
@@ -38,6 +38,7 @@ const (
 	markerName = "controllertools:marker:generateHelp"
 	header     = `
 // +build !ignore_autogenerated
+// +build !skip
```

Comment:
@NikhilSharmaWe:

If we skip the build files, the tests will fail:

```
r$ go test ./... -cover -tags skip
?   	sigs.k8s.io/kubebuilder/testdata/project-v3	[no test files]
# sigs.k8s.io/kubebuilder/testdata/project-v3/api/v1
api/v1/zz_generated.deepcopy.go:2: +build lines do not match //go:build condition
FAIL	sigs.k8s.io/kubebuilder/testdata/project-v3/api/v1 [build failed]
ok  	sigs.k8s.io/kubebuilder/testdata/project-v3/controllers	11.584s	coverage: 0.0% of statements
FAIL
```

So, I do not think we can address the request made in the Kubebuilder because of the coverage. 
In the comment: https://github.com/kubernetes-sigs/kubebuilder/pull/2668#issuecomment-1119482191, I explained how/where/when it is and if we wish to change what is the process. 

But I do not think that we should address this change here. 
Let's close this one and see how can we address/help who asked it in the Kubebuilder side?